### DISCUSSION TITLE
Crashes in Silver Seed

### DISCUSSION THREAD

**Discussion by schroeders-welt** [exult/exult#527] [General]
Hello there. I started a new game of Serpent Isle on a completely new installation of the latest Exulz version (1.8.0). So far no problems in the main game. Now, after I started the Silver Seed extension, the Exult application crashes to Desktop after entering combat mode in the game.
Stderr says:

```
Could not open file '.\forgeofvirtue\static\mainshp.flx
Could not open file '.\silverseed\static\mainshp.flx
Could not open file '.\serpentbeta\static\mainshp.flx
Warning, trailing slash in path: "<BLACKGATE_STATIC>\"
Warning, trailing slash in path: "<SERPENTISLE_STATIC>\"
SDL Reports 640x400 16 bpp and 32 bpp windowed surfaces are not OK. Windowed scalers may not work properly.
terminate called after throwing an instance of 'std::bad_weak_ptr'
  what():  bad_weak_ptr
```
Is anyone familiar with this error? Makes the game more or less unplayable.

**Comment by DominusExult** [comment]
Please try our git master as this sounds like a problem we fixed since the 1.8 release

**Comment by schroeders-welt** [comment]
Problem is gone with latest Build 1.9. Thank you.

**Comment by DominusExult** [comment]
Thank you for getting back!
